ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  BalanceViewController.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 12/03/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+import SwiftKeychainWrapper
+import SnapKit
+
+class BalanceViewController: BaseViewController {
+    private let viewModel: BalanceViewModel
+    private let rm: RequestManager
+    
+    weak var screenStackView: UIStackView!
+    weak var balanceLabel: UILabel!
+    weak var balanceTitle: UILabel!
+    weak var reloadButton: UIButton!
+    weak var accountButton: UIButton!
+    
+    override init() {
+        self.viewModel = BalanceViewModel()
+        self.rm = RequestManager()
+        super.init()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.view.backgroundColor = UIColor.Theme.backgroundColor
+        
+        let screenStackView = UIStackView()
+        screenStackView.axis = .vertical
+        screenStackView.spacing = 60
+        self.screenStackView = screenStackView
+        self.view.addSubview(screenStackView)
+        
+        screenStackView.snp.makeConstraints { make in
+            make.centerX.centerY.equalToSuperview()
+            make.leading.greaterThanOrEqualTo(20)
+            make.trailing.lessThanOrEqualTo(-20)
```

Comment:
Tady ti zůstal stejnej problém jen na vertikální ose 😄 Jak jseš si jistej, že to nepřeteče obrazovku? 😏

Reply:
Dnes ráno jsem si toho všiml a v nejnovější branchi to opravil. Letím to opravit v tomto PR.

Reply:
Tak na to házej bobek, jestli je to fixed někde dál, tak ok.

Reply:
Opraveno

Reply:
Jo počkat, až teď mi došlo, co myslíš. Na vertikální osu jsem nějak nespoléhal, když mám pro appku nastavený jen portrait mode.

Reply:
No a o tom mluvím 🙂 Na to nespoléhej